src/services/sync/strategies/consumerActions/UserAction.js

```diff
@@ -26,17 +26,39 @@ class UserAction extends BaseConsumerAction {
 				schoolDn: user.schoolDn,
 				systemId: user.systemId,
 			});
-		} else if (!school.inMaintenance) {
-			const foundUser = await UserRepo.findByLdapIdAndSchool(user.ldapId, school._id);
-			if (foundUser !== null) {
-				await this.updateUserAndAccount(foundUser, user, account);
-			} else {
-				await this.createUserAndAccount(user, account, school._id);
-			}
+		}
+
+		if (school.migrationMode) {
```

Comment:
where does this come from? 
```suggestion
		if (school.inUserMigration) {
```